src/patchelf.cc

```diff
@@ -1544,6 +1556,13 @@ void ElfFile<ElfFileParamNames>::replaceNeeded(const std::map<std::string, std::
         debug("found .gnu.version_r with %i entries, strings in %s\n", verNeedNum, versionRStringsSName.c_str());
 
         unsigned int verStrAddedBytes = 0;
+        // it may be that is is .dynstr again, in which case we must take the already
```

Comment:
```suggestion
        // it may be that is  .dynstr again, in which case we must take the already
```

Reply:
That should have been "it is", thanks for spotting.